.gitignore

```diff
@@ -41,31 +28,6 @@ bld/
 # Visual Studio 2017 auto generated files
 Generated\ Files/
```

Comment:
It is also Visual Studio related content

Reply:
yes, i should be here, because of development ide